index.html

```diff
@@ -664,10 +682,31 @@ <h3 class="modal-title">
                localStorage.setItem('showGlobal', '');
              }
              localStorage.setItem('showPasses', '');
+             localStorage.setItem('showReviewed', '');
              localStorage.setItem('showVisited', $scope.showVisited);
              location.reload();
           }
 
+         // Get reviewed options from localStorage
+         $scope.showReviewed = localStorage.getItem('showReviewed');
+
+         if (showReviewed && showReviewed !=='all') {
+            $scope.allData = JSON.parse(JSON.stringify(allData)).filter(val => val.resortReview.score);

```

Comment:
I think this will filter out any ski resorts that Johnathan ranks as a 0. But I doubt that will happen